samples/ext/nffs/README.rst

```diff
@@ -0,0 +1,40 @@
+.. _nffs_sample:
+
+File system: NFFS
+####################
+
+Overview
+*********
+Set up a Newtron Flash File system (NFFS) on a Nordic Semiconductor NRF5x device.
+
+
+Requirements
+************
+* NRF5x development kit. (Has only been tested on boards: BOARD=nrf52_pca10040 and BOARD=nrf52840_pca10056).
+* A serial terminal for observing example output. 115200 baud, no parity, 8 data bits, 1 stop bit, no HW flow control.
+
```

Comment:
Add information about installing the nRF5x Command Line Tools (in particular, ``nrfjprog``) as a requirement. It looks like they're included when installing the SEGGER software, but you should clarify that here.  I found [this Nordic material](http://infocenter.nordicsemi.com/index.jsp?topic=%2Fcom.nordic.infocenter.tools%2Fdita%2Ftools%2Fnrf5x_command_line_tools%2Fnrf5x_installation.html) that might be worth linking to as well.